.travis.yml

```diff
@@ -10,35 +10,20 @@ cache:
 
 language: ruby
 rvm:
-  - 1.8.7
-  - 1.9.3
```

Comment:
<img width="1006" alt="screen shot 2017-03-19 at 22 21 06" src="https://cloud.githubusercontent.com/assets/179534/24084882/6e0abba2-0cf2-11e7-894b-f3ad4c2d5a7a.png">

> I prefer not to break it if I don't have to.

From my POV it is broken. I'm not insisting here. Just thinking out loud.